src/main/java/com/microsoft/azure/relay/TimeoutHelper.java

```diff
@@ -9,11 +9,11 @@
 	private boolean deadlineSet;
 	private Duration originalTimeout;
 
-	protected TimeoutHelper(Duration timeout) {
+	TimeoutHelper(Duration timeout) {
```

Comment:
>TimeoutHelper [](start = 1, length = 13)

You don't need to change anything now, but I believe the philosophy with public access on types which are internal is to imagine if the type were made public and someone in another package was going to use the class. What should that "public" API look like? In that hypothetical scenario would they need to call the constructor and some of these methods? Probably.

Again no change needed for now, just food for thought next time implementing an internal class.

Reply:
Maybe these didn't follow the rule I'm describing in the original C# version. If the were internal there that would explain.

---
In reply to: [257346026](https://github.com/Azure/azure-relay-java/pull/14#discussion_r257346026) [](ancestors = 257346026)